nl/guides/exercises/creating-exercises/content/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Oefeningen opstellen: leesactiviteit"
```

Comment:
I am not sure if I would put this here. Perhaps this would be better as a separate page under the "Soorten oefeningen" heading, or even put it a level higher.

Reply:
I think it's not that important to make it a level higher and it still belongs to the tutorial. I see it as a subset of an exercise. I just want to show the user everything they learned also applies to reading activities. It's not an exercise, so I wouldn't categorize as such.

Reply:
I agree with Niko here. This fits best in the "soorten oefeningen" section.